### PR TITLE
Documentation: Fix OIDC credentials reference to secret key

### PR DESCRIPTION
**Pull request by michalvavrik**
@fedinskiy mentioned `secret.key` does not exist anywhere in `quarkus.oidc.credentials.*` namespace. I realize it is common to use relative config key names in the `io.quarkus.oidc.common.runtime.OidcCommonConfig` and also it uses term _secret key_, but I don't think it's clear enough to use relative path for config key from parent config group, hence using absolute config property name.

### CODE REVIEW

File: extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java [hunk 1]
```diff
@@ -180,7 +180,7 @@ public static enum Method {
             }
 
             /**
-             * The client secret value - it will be ignored if 'secret.key' is set
+             * The client secret value - it will be ignored if 'quarkus.oidc.credentials.secret' is set
```
Comment (sberyozkin) [inline extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java:183]: @michalvavrik The problem here is that it wont work with `quarkus-oidc-client`, so somehow we should mention both options

Comment (michalvavrik) [reply extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java:183]: okay, I didn't realize that. But if I couldn't figure this after looking into code for several minutes, I'm not sure how could users be expect to understand that. I'll think about it...

Comment (michalvavrik) [reply extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java]: What about `credentials.secret`? it's not perfect, but at least it exists.

I pushed that change, should you have better idea, please let me know.

Comment (sberyozkin) [reply extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java]: @michalvavrik sounds good

**Review by sberyozkin — APPROVED** [review]

**Pull request merged**